validation.rst

```diff
@@ -489,8 +489,8 @@ options can be specified in this way.
 This is purely meant to make the configuration of the most common option of
 a constraint shorter and quicker.
 
-If you're ever unsure of how to specify an option, either check the API documentation
-for the constraint or play it safe by always passing in an array of options
+If you're ever unsure of how to specify an option, either check :namespace:`the API documentation
+for the constraint <Symfony\Component\Validator\Constraints>` or play it safe by always passing in an array of options
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
I think I followed synthax for links to Api.
https://github.com/symfony/symfony-docs/blob/master/contributing/documentation/format.rst#adding-links

Reply:
Like that for namespaces =>  :namespace:`Symfony\\Component\\BrowserKit`
And like that, if you want to use a title =>  :doc:`Spooling Email </email/spool>`